x-pack/plugins/aiops/public/components/explain_log_rate_spikes/explain_log_rate_spikes_content/explain_log_rate_spikes_content.tsx

```diff
@@ -0,0 +1,170 @@
+
+import type { DataView } from '@kbn/data-views-plugin/public';
+import type { Dictionary } from '@kbn/ml-url-state';
+import type { WindowParameters } from '@kbn/aiops-utils';
+import type { SignificantTerm } from '@kbn/ml-agg-utils';
+
+import type { Moment } from 'moment';
+import { useData } from '../../../hooks/use_data';
+import { DocumentCountContent } from '../../document_count_content/document_count_content';
+import { ExplainLogRateSpikesAnalysis } from '../explain_log_rate_spikes_analysis';
+import type { GroupTableItem } from '../../spike_analysis_table/types';
+import { useSpikeAnalysisTableRowContext } from '../../spike_analysis_table/spike_analysis_table_row_provider';
+
+const DEFAULT_SEARCH_QUERY = { match_all: {} };
+
+export function getDocumentCountStatsSplitLabel(
+  significantTerm?: SignificantTerm,
+  group?: GroupTableItem
+) {
+  if (significantTerm) {
+    return `${significantTerm?.fieldName}:${significantTerm?.fieldValue}`;
+  } else if (group) {
+    return i18n.translate('xpack.aiops.spikeAnalysisPage.documentCountStatsSplitGroupLabel', {
+      defaultMessage: 'Selected group',
+    });
+  }
+}
+
+export interface ExplainLogRateSpikesContentProps {
+  /** The data view to analyze. */
+  dataView: DataView;
+  setGlobalState?: (params: Dictionary<unknown>) => void;
+  /** Timestamp for the start of the range for initial analysis */
+  initialAnalysisStart?: number;
+  timeRange?: { min: Moment; max: Moment };
+  /** Elasticsearch query to pass to analysis endpoint */
+  esSearchQuery?: estypes.QueryDslQueryContainer;
+}
+
+export const ExplainLogRateSpikesContent: FC<ExplainLogRateSpikesContentProps> = ({
+  dataView,
+  setGlobalState,
+  initialAnalysisStart,
+  timeRange,
+  esSearchQuery = DEFAULT_SEARCH_QUERY,
+}) => {
+  const [windowParameters, setWindowParameters] = useState<WindowParameters | undefined>();
+
+  const {
+    currentSelectedSignificantTerm,
+    currentSelectedGroup,
+    setPinnedSignificantTerm,
+    setPinnedGroup,
+    setSelectedSignificantTerm,
+    setSelectedGroup,
+  } = useSpikeAnalysisTableRowContext();
+
+  const { documentStats, earliest, latest } = useData(
+    dataView,
+    'explain_log_rage_spikes',
+    esSearchQuery,
+    setGlobalState,
+    currentSelectedSignificantTerm,
+    currentSelectedGroup,
+    undefined,
+    timeRange
+  );
+
+  const { sampleProbability, totalCount, documentCountStats, documentCountStatsCompare } =
+    documentStats;
+
+  function clearSelection() {
+    setWindowParameters(undefined);
+    setPinnedSignificantTerm(null);
+    setPinnedGroup(null);
+    setSelectedSignificantTerm(null);
+    setSelectedGroup(null);
+  }
+
+  return (
+    <EuiResizableContainer style={{ height: '400px' }} direction="vertical">
+      {(EuiResizablePanel, EuiResizableButton) => (
+        <>
+          <EuiResizablePanel
+            mode={'collapsible'}
+            initialSize={60}
```

Comment:
This panel is always getting a scrollbar for me. Does the `initialSize` or `minSize` need to be larger?